### PR TITLE
chore: fix download paths

### PR DESCRIPTION
**Pull request by sebmos**

### CODE REVIEW

File: packages/client/install/README.md
Comment (sebmos) [inline packages/client/install/README.md:1]: The link to Finsemble UAT (https://storage.googleapis.com/reactive-trader-finsemble-uat/pkg/ReactiveTraderFinsemble.exe) isn't right, either.

File: README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ Please see [our Showcase page](https://weareadaptive.com/showcase/) for a full l
 [finsemble]: https://cosaic.io/finsemble/
 [storybook]: https://www.reactivetrader.com/storybook
 [style guide]: https://www.reactivetrader.com/styleguide
-[openfin-win]: ./install/Reactive-Launcher-Demo.exe?raw=true
```
Comment (algreasley) [inline README.md:30]: I've half a mind to remove these .. and just point to the install README ..

File: packages/client/install/README.md
Comment (algreasley) [reply packages/client/install/README.md:1]: yep, correct address is https://storage.googleapis.com/reactive-trader-finsemble-uat/pkg/ReactiveTraderFinsemble-UAT.exe .. you ok to update this commit?

File: README.md (hunk 1 above)
Comment (sebmos) [reply README.md:30]: Yes, done

**Review by algreasley — APPROVED** [review]

**Comment by algreasley** [comment]
thanks @sebmos

**Pull request merged**